frontend/src/components/__tests__/ColumnChartPreview.test.tsx

```diff
@@ -3,6 +3,8 @@ import { render } from "@testing-library/react";
 import { MemoryRouter } from "react-router-dom";
 import ColumnChartPreview from "../ColumnChartPreview";
 
+jest.mock("../../hooks");
```

Comment:
Hooks are actually automaticallly mocked because the `setupTests.ts` file already does it. This line doesn't affect but it shouldn't be necessary.